### PR TITLE
Endpoints for LLM, Embedder and Plugin settings

### PR DESCRIPTION
**Pull request by zAlweNy26**
# Description

I added:
- One endpoint for LLM settings to get the settings and schema of a specific LLM
- One endpoint for Embedder settings to get the settings and schema of a specific Embedder
- One endpoint for Plugin settings to get the list of all the settings and schemas of the available plugins

I also moved the endpoints to a more generic name, so that in future we can add other endpoints related to them.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: core/cat/routes/memory.py [hunk 1]
```diff
@@ -151,42 +137,57 @@ async def wipe_single_collection(request: Request, collection_id: str = "") -> D
...
-        "deleted": to_return,
+        "deleted": True,
     }
 
-#DELETE conversation history from working memory
-@router.delete("/working-memory/conversation-history/")
-async def wipe_conversation_history(
+
+# DELETE memories
+@router.delete("/{collection_id}/point/{memory_id}/")
```
Comment (zAlweNy26) [inline core/cat/routes/memory.py:157]: @pieroit would you prefer like that or directly `/{collection_id}/{memory_id}/` ?

Comment (pieroit) [reply core/cat/routes/memory.py:157]: let's leave `point`, in the future we may consider other aspects of a collection as a resource

Comment (zAlweNy26) [reply core/cat/routes/memory.py:157]: Ok, perfect!

Comment (pieroit) [reply core/cat/routes/memory.py:157]: thanks for fixing the tests!!!
Going for `/collections/{collection_id}/points/{memory_id}/` which should be the most RESTful

**Comment by pieroit** [comment]
Closing this PR because I cannot rebase it to branch `zAlweNy26-specific_llm-embedder-plugin_settings`

Your commits are saved and part of the history as you can see [here](https://github.com/cheshire-cat-ai/core/commits/zAlweNy26-specific_llm-embedder-plugin_settings), so they will end up in `develop` and then `main`.

Thanks :dancers:

**Pull request closed without merging**